.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   rules: {
     'no-console': 'error',
     'no-debugger': 'error',
+    'no-template-curly-in-string': 0,
```

Comment:
Is there a reason to disable this check globally instead on a line-by-line base?

Reply:
> (**) We may need to disable no-template-curly-in-string eslint rule

This rule generally prevents using interpolation in `nuxt.config`

Reply:
Whoops, missed that. Thanks!